content/chat/setup.textile

```diff
@@ -124,13 +124,19 @@ blang[react].
 blang[javascript,swift,kotlin].
 
 ```[javascript]
+import { ChatClientOptions, LogLevel } from '@ably/chat';
+
 const realtimeClient = new Ably.Realtime({ key: '{{API_KEY}}', clientId: '<clientId>'});
-const chatClient = new ChatClient(realtimeClient);
+const chatClientOptions: ChatClientOptions = { logLevel: LogLevel.Error }
+const chatClient = new ChatClient(realtimeClient, chatClientOptions);
```

Comment:
I think it's nice to pass options so copy-pasting results in code that's easy to adjust, but should we keep it short?
```suggestion
const chatClient = new ChatClient(realtimeClient, { logLevel: LogLevel.Error });
```